### PR TITLE
Adding to the status_info

### PR DESCRIPTION
**Pull request by clhliu**
- Checking if I'm using pull request correctly
- Checking if I did the function properly

### CODE REVIEW

File: risk/commands.py [hunk 1]
```diff
@@ -9,6 +9,20 @@ def help_info(player, game_master):
...
+	territories = game_master.player_territories(player)
+	cavalries   = 0
+	infantries  = 0
+	artilleries = 0
+	for territory in territories:
+		cavalries   += territory.cavalries
+		infantries  += territory.infantries
+		artilleries += territory.artilleries
+		print '%s:Cavlries: %sInfantries: %sArtilleries: %s' % (territory, calvaries, infantries, artilleries)
+		cavalries   = 0
```
Comment (frgorp) [inline risk/commands.py]: why do we need to reset the variables?

File: risk/commands.py [hunk 2]
```diff
@@ -9,6 +9,20 @@ def help_info(player, game_master):
...
+	print 'Territories:'
+	territories = game_master.player_territories(player)
+	cavalries   = 0
+	infantries  = 0
+	artilleries = 0
+	for territory in territories:
+		cavalries   += territory.cavalries
+		infantries  += territory.infantries
+		artilleries += territory.artilleries
+		print '%s:Cavlries: %sInfantries: %sArtilleries: %s' % (territory, calvaries, infantries, artilleries)
```
Comment (frgorp) [inline risk/commands.py]: This works, but it's very hard to extract the necessary information, a better way to allow users to get an idea "at a glance" would be to format the output to something like:

[south america]
Cavalries: 5
Infantries: 5
Artilleries: 5

**Comment by frgorp** [comment]
I've added some comments on your changes, please have a look and revise.

File: risk/commands.py [hunk 3]
```diff
@@ -9,6 +9,20 @@ def help_info(player, game_master):
...
 def status_info(player, game_master):
     print 'status'
+	print 'Player %s:\n' % player 
+	print 'Territories:'
+	territories = game_master.player_territories(player)
+	cavalries   = 0
+	infantries  = 0
+	artilleries = 0
+	for territory in territories:
+		cavalries   += territory.cavalries
```
Comment (frgorp) [inline risk/commands.py]: These are tabs!

**Comment by frgorp** [comment]
status_info command contains tabs and is crashing the game on start.

File: risk/commands.py (hunk 1 above)
Comment (clhliu) [inline risk/commands.py]: I thought it was suppose to print out the amount of cavalries, infantries, and artillery for each territory.

Comment (frgorp) [inline risk/commands.py]: yes, but think about it, this code doesn't really make sense and is kind of wasteful. On one hand, you are calculating the _total_ amount of armies but then you throw away the results at the end of every loop.

**cavalries += territory.cavalries** really serves no purpose, you might as well just use **territory.cavalries** and get rid of all the assignments.

On the other hand, you _might_ want to display the **total** amount of armies a player owns, in which case you can rewrite the loop as:

```
total_cavalries = 0
total_infantries = 0
total_artilleries = 0
for territory in territories:
    total_cavalries   += territory.cavalries
    total_infantries  += territory.infantries
    total_artilleries += territory.artilleries
    print '%s:Cavlries: %sInfantries: %sArtilleries: %s' % (territory, territory.calvaries, territory.infantries, territory.artilleries)
```

# print grand total here

File: risk/commands.py [hunk 4]
```diff
@@ -9,6 +9,17 @@ def help_info(player, game_master):
     
 def status_info(player, game_master):
     print 'status'
+	print 'Player %s:\n' % player
```
Comment (frgorp) [inline risk/commands.py]: can you recheck this? It's still showing up as tabs, Python doesn't like it when you mix and match tabs with spaces. Mine crashes with:

```
Traceback (most recent call last):
  File "./risk.py", line 8, in <module>
    import risk.game_master
  File "/home/ffred/workspace/risk/risk/game_master.py", line 2, in <module>
    import risk.commands
  File "/home/ffred/workspace/risk/risk/commands.py", line 12
    print 'Player %s:\n' % player 
    ^
IndentationError: unexpected indent
```

Be sure to run risk.py in the root directory to test out your changes!

File: risk/commands.py [hunk 5]
```diff
@@ -9,6 +9,17 @@ def help_info(player, game_master):
...
+	print 'Territories:'
+	territories = game_master.player_territories(player)
+ 	cavalries   = 0
+ 	infantries  = 0
+ 	artilleries = 0
+ 	for territory in territories:
+		cavalries   = territory.cavalries
+		infantries  = territory.infantries
+		artilleries = territory.artilleries
+		print '[%s]:Cavlries: %sInfantries: %sArtilleries: %s' % (territory, calvaries, infantries, artilleries)
```
Comment (frgorp) [inline risk/commands.py]: you might want to add newlines between each piece of information to make it easier to digest, something like:
    print "[%s]\n" \
           "Cavalries: %s\n" \
           "Infantries: %s\n" \
           "Artilleries: %s\n" % (territory, cavalries, infantries, artilleries)

**Pull request merged**